omnigan/generator.py

```diff
@@ -219,6 +191,7 @@ def __init__(self, opts):
             res_norm=opts.gen.h.res_norm,
             activ=opts.gen.h.activ,
             pad_type=opts.gen.h.pad_type,
+            output_activ="sigmoid",
```

Comment:
should we have that though? shouldn't we have an unbounded linear output?

Reply:
Yes probably, but we're not using height anyways for the moment

Reply:
true, it just seemed like you changed that line, that's why I asked :)